pkg/integration/tests/interactive_rebase/move_with_custom_comment_char.go

```diff
@@ -0,0 +1,84 @@
+package interactive_rebase
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var MoveWithCustomCommentChar = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Directly move a commit all the way down and all the way back up with a custom comment char",
```

Comment:
just confirming that this test fails under the old code and passes under the new code?

Reply:
The former. Not sure I understand what you are asking in the latter...